### ISSUE TITLE
Sauce Code Pro Italic Nerd Font Complete Variants Are Broken in v2.3.3

### ISSUE THREAD

**Issue by fgimian** [ryanoasis/nerd-fonts#1139]
#### 🗹 Requirements

- [x] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [x] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [x] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 All 4 Sauce Code Pro Italic Nerd Font Complete variants appear to be not valid font files on Windows since v2.3.3

Experienced behavior:
Attempting to open or install any of the italic fonts mentioned results in the following error on Windows 11 Pro:

![image](https://user-images.githubusercontent.com/1811813/222942933-ea2c0b9f-d8cf-4675-bf91-e686c9408262.png)

This works correctly with the previous release v2.3.2.

Expected behavior:
The font should open and install without error.

Example symbols:
N/A

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_
   - Sauce Code Pro Italic Nerd Font Complete.ttf
   - Sauce Code Pro Italic Nerd Font Complete Mono.ttf
   - Sauce Code Pro Italic Nerd Font Complete Mono Windows Compatible.ttf
   - Sauce Code Pro Italic Nerd Font Complete Windows Compatible.ttf
- _Where did you get the file from (download link, self patched, source downloaded from link...)_
   - Downloaded directly from GitHub Releases
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_
   - Windows Terminal
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_
   - Windows 11 Pro 21H2

#### ★ Screenshots (Optional)

Included above 😄 

Thanks heaps
Fotis

**Comment by Finii** [comment]
Sorry to hear that. Thanks for reporting.

<img width="643" alt="Screenshot 2023-03-05 at 12 47 02" src="https://user-images.githubusercontent.com/16012374/222958634-f4b1181c-032f-4829-86c4-df30aab8c7bd.png">

**Comment by Finii** [comment]
Fontforge on Linux is fixing something ;-)

![Screenshot from 2023-03-05 12-51-37](https://user-images.githubusercontent.com/16012374/222958877-864f1174-dc9d-4404-af24-d7c1f1091f7f.png)

**Comment by Finii** [comment]
Ah, yes, ... I remember

![image](https://user-images.githubusercontent.com/16012374/222959102-0c710363-a4f2-47ef-9d1f-bb20684e2553.png)

We have (had) 2 italic fonts that come out with the same name and are garbled because they are produced in parallel, writing to the same one output file. Lets see...


This is a duplicate of
* #1084

and has been fixed with
* #1085 

Please use the files directly out of the repo until next release (see bottom of https://github.com/ryanoasis/nerd-fonts/issues/1084#issuecomment-1406146906)

**Comment by fgimian** [comment]
> Ah, yes, ... I remember
> 
> ![image](https://user-images.githubusercontent.com/16012374/222959102-0c710363-a4f2-47ef-9d1f-bb20684e2553.png)
> 
> We have (had) 2 italic fonts that come out with the same name and are garbled because they are produced in parallel, writing to the same one output file. Lets see...
> 
> This is a duplicate of
> 
>     * [Sauce Code Pro Italic Nerd Font Complete (Mono) can't be opened on Windows #1084](https://github.com/ryanoasis/nerd-fonts/issues/1084)
> 
> 
> and has been fixed with
> 
>     * [SourceCodePro: Remove obsolete font #1085](https://github.com/ryanoasis/nerd-fonts/pull/1085)
> 
> 
> Please use the files directly out of the repo until next release (see bottom of [#1084 (comment)](https://github.com/ryanoasis/nerd-fonts/issues/1084#issuecomment-1406146906))

Thank you so much for your help and apologies that I missed the existing issue 😄

**Comment by Finii** [comment]
Release is in repo, release as packages pending.
